packages/hash/design-system/yarn.lock

```diff
@@ -0,0 +1,2387 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
There should only be one `yarn.lock` at the top of the repo. Looks like this one was added by accident before `design-system` became a yarn workspace.

Reply:
Thanks for catching!